app/models/crosswalk_issue.rb

```diff
@@ -38,9 +43,20 @@ def self.rebuild
           AND crosswalks.ope IS NULL
         )
         AND UPPER(weams.campus_type) is distinct from 'E'
+      UNION
+      SELECT
+        NULL,
+        crosswalks.id,
+        ipeds_hds.id,
+        '#{IPEDS_ORPHAN_TYPE}'
+      FROM ipeds_hds
+        LEFT OUTER JOIN crosswalks ON ipeds_hds.cross = crosswalks.cross OR ipeds_hds.ope = crosswalks.ope
+      WHERE crosswalks.ID IS NULL
     SQL
 
     InstitutionProgram.connection.execute(sql)
   end
   # rubocop:enable Metrics/MethodLength
+
+  scope :by_issue_type, ->(n) { where(issue_type: n) }
```

Comment:
Nitpick: can you move this toward the top of the class?  Maybe right below the `belongs_to` declarations. Typically that's where you [find these](https://github.com/rubocop-hq/rails-style-guide#macro-style-methods).